code/game/machinery/cloning.dm

```diff
@@ -230,7 +230,8 @@
 	if(!R.dna)
 		R.dna = new /datum/dna()
 
-	var/mob/living/carbon/human/H = new /mob/living/carbon/human(src, R.dna.species)
+	var/mob/living/carbon/human/H = new /mob/living/carbon/human(src)

```

Comment:
make sure to look for this elsehwere; pretty sure others utilize this ability.

Reply:
I looked over every human/New call I could find but I didn't see anything else -- should I readd my debugging message to human/New to throw warnings if anything calls it with more than a loc argument?

Reply:
Yeah, I would, *just* in case.

Reply:
Done

Reply:
I knew there was another one---there's one under monkey cubes.